From 931079d1362aba4cacda7afcbb63b9a15fde7083 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 29 Jan 2021 11:21:01 +0100
Subject: [PATCH] Revert "Change default key_type in muacme.conf to EC"

Keep the original defaults to avoid risk of breaking change in Alpine 3.13.

This reverts commit 4b8d4e63e1549dc947787bca190a21df0764ae54.
---
 muacme.conf | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/muacme.conf b/muacme.conf
index f72dde0..97089bf 100644
--- a/muacme.conf
+++ b/muacme.conf
@@ -22,15 +22,15 @@
 # number of days.
 #days=37
 
-# Key bit length (defaults to 3072). Only applies to newly generated keys.
-# RSA key length must be a multiple of 8 between 2048 and 8192.
+# Key bit length (default 2048 for RSA, 256 for EC). Only applies to newly
+# generated keys. RSA key length must be a multiple of 8 between 2048 and 8192.
 # EC key length must be either 256 (NID_X9_62_prime256v1 curve) or
 # 384 (NID_secp384r1 curve).
-key_bits=256
+#key_bits=3072
 
-# Key type, either "RSA" (default) or "EC". Only applies to newly generated
-# keys. The bit length can be specified with $key_bits.
-key_type="EC"
+# Key type, either "RSA" or "EC". Only applies to newly generated keys.
+# The bit length can be specified with $key_bits.
+#key_type="RSA"
 
 # Path of the renew hook program that is executed after at least one
 # certificate has been successfully renewed. It will get domain names of the
-- 
2.16.4

